packages/error-reporting/CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# How to become a contributor and submit your own code
```

Comment:
Is the global one useful instead? https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/.github/CONTRIBUTING.md -- That is the one that will appear when a user goes to open a new issue. If it needs to be expanded or worked with to make it more general, feel free to edit it.